src/components/Modal/CreateKeywordModal.jsx

```diff
@@ -0,0 +1,160 @@
+import { useState } from "react";
+
+import { ERROR_MESSAGE, MODAL_TYPE } from "../../config/const";
+import CreateKeywordButton from "../Button/CreateKeywordButton";
+import SelectGroupDropDown from "../DropDown/SelectGroupDropDown";
+import PlusIcon from "../Icon/PlusIcon";
+import Button from "../UI/Button";
+import Label from "../UI/Label";
+import ModalBackground from "./ModalBackground";
+import ModalFrame from "./ModalFrame";
+import ModalMount from "./ModalMount";
+
+const CreateKeywordModal = () => {
+  const [selectedGroup, setSelectedGroup] = useState("");
+  const [inputValue, setInputValue] = useState({
+    newGroup: "",
+    keyword: "",
+  });
+  const [errorMessage, setErrorMessage] = useState({
+    newGroup: "",
+    keyword: "",
+  });
+  const [isCreateNewGroup, setIsCreateNewGroup] = useState(false);
```

Comment:
```suggestion
  const [isCreatingNewGroup, setIsCreatingNewGroup] = useState(false);
  const [selectedGroup, setSelectedGroup] = useState("");
  const [inputValue, setInputValue] = useState({
    newGroup: "",
    keyword: "",
  });
  const [errorMessage, setErrorMessage] = useState({
    newGroup: "",
    keyword: "",
  });
```
1. `isCreateNewGroup` 상태명을 영문법에 맞게 `isCreatingNewGroup`로 수정해보면 어떨까요? Boolean으로써 `is`로 시작되는 형태는 유지하되, 동사가 2개인 점을 고려해보았어요. 
2. 유저 스토리상 "새 그룹 생성 여부"`isCreateNewGroup`이 기점이 되다보니, 코드로도 가장 상단으로 이동시키면 흐름 이해가 더 수월할 것 같습니다. 현재 순서도 문제가 되는 것은 아니지만, 가독성을 고려하여 제안 드려봅니다! :)

Reply:
제안 주신대로 적용하였습니다 :)